### DISCUSSION TITLE
[Layout] Columns vs Tiles

### DISCUSSION THREAD

**Discussion by kyledurand** [Shopify/polaris-react#7756] [Polls]
Columns and tiles have the same interface and the same css. They are serving the same purpose and causing confusion around 'when to use' documentation. At this point we can't recommend one over the other because they do exactly the same things

Columns interface | Tiles interface
---|---
![image](https://user-images.githubusercontent.com/6844391/202554125-c0739a04-8caa-48be-ad5b-84f5163da3df.png) | ![image](https://user-images.githubusercontent.com/6844391/202554203-8dfd23ad-3209-4a16-bc09-b204dc5908b7.png)

Columns css | Tiles css
---|---
![image](https://user-images.githubusercontent.com/6844391/202554472-830d8c65-ff1b-426f-918d-3c43acf80d65.png) | ![image](https://user-images.githubusercontent.com/6844391/202554531-98974eab-e0a3-4c98-b442-92d88262c6c4.png)

![image](https://user-images.githubusercontent.com/6844391/202543043-b53f3499-d735-4c92-a548-9f53abec9194.png)

**Comment by laurkim** [comment]
We huddled and determined since Columns supports the same functionality that Tiles does, we are removing Tiles in favor of Columns. 

[Issue](https://github.com/Shopify/polaris/issues/7770) to track removal.

**Comment by laurkim** [comment]
The [PR](https://github.com/Shopify/polaris/pull/7775) to remove `Tiles` has been merged.
